docs/source/api/python/techniques/PearlPowderDiffractionISIS-v1.rst

```diff
@@ -0,0 +1,128 @@
+===============================
+
+Overview
+--------
+
+A wide variety of algorithms are available within Mantid for
+processing of data, writing results files, etc.  These may be run
+interactively, one at a time, but in general scripting (using the
+Python scripting language) enables a series of algorithms to be called
+in a specific order to carry out more complex tasks, such as data
+normalisation and writing data files suitable for input to Rietveld
+refinement programmes (GSAS, FullProf, etc).
+The script used to process Pearl data have been integrated to Mantid
+which can be found inside the following directory on a Windows machine:
+`'C:\\\MantidInstall\\scripts\\PearlPowderISIS'`.
+
+Further extensions can be expected for next releases as it is under
+active development and feedback is very much welcome. The sections
+below describes how the script can be ran and help user develop
+understanding of the script. By utilising this script it is assumed
+that the scientists/users at ISIS have permission to read and write
+from the network. Where **`P:\\\ `** drive is linked to **`\\\\britannic\\Pearl\\`**
+and **`X:\\\ `** drive is linked to **`\\\\isis\\inst$\\NDXPEARL\\Instrument`**.
+
+Run Requirement
+---------------
+
+A simple script is required to be ran inside the `Scripting Window
+<http://docs.mantidproject.org/nightly/interfaces/ScriptingWindow.html>`_
+on mantid in order to carry out the data normalisation.
+
+The script which is written inside `Scripting Window <http://docs.
+mantidproject.org/nightly/interfaces/ScriptingWindow.html>`_ on Mantid
+requires the following from the user (followed up with an example
+usage):
+
+- user name e.g: (`'Bull'`)
+- cycle number e.g: (`'15_4'`)
+- run number/s e.g: (`'92476_92479'`)
+- fmode e.g: (`'trans'`)
+- ttmode e.g: (`'TT70'`)
+
+If the following parameters are not passed, the default values are
+used instead.
+
+- ext e.g: (`'raw'`, `'s01'`) - file extension, default is set as `raw`
+- atten e.g: (`True` or `False`) - whether use attentuation file, default
+  is set as `True`
+- debug e.g: (`True`or `False`) - whether to use debug mode, default
+  is set as `False`
+
+- optional: Attentuation file directory - if you would wish to use
+  a different Attentuation file e.g:
+  (`'P:\\Mantid\\Attentuation\\PRL112_DC25_10MM_FF.OUT'`)
+- optional: Output directory - only if you wish to write out the
+  files to preferred directory/location instead of the Pearl network.
+  e.g: (`'C:\\Mantid\\MantidOut\\'`)
+
+*Usage examples can be found under the* :ref:`pearl-powder-diffraction-usage-ref` *section.*
+
+File & Folders
+--------------
+
+The routines/script has been differentiated from the list of
+directories of `calibration` and `raw` files. The calibration
+directories can be found within `'pearl_calib_factory.py'`,
+whereas the the raw directories can be found within
+`'pearl_cycle_factory.py'`.
+
+You are not require to set calibration and raw files directory,
```

Comment:
typo? require->required.
Can be fixed later on.
Once we have more system tests in place, a list of the files that are normally produced by the `focus` routine would be useful next to this section.